python/pfs/drp/stella/fitPfsFluxReference.py

```diff
@@ -154,10 +157,16 @@ class FitPfsFluxReferenceConfig(PipelineTaskConfig, pipelineConnections=FitPfsFl
         default="psf",
         optional=False,
     )
+    minimizationMethod = Field(
```

Comment:
Use `ChoiceField` to limit the possible values?

Reply:
I don't want this parameter to be restricted to "brute-force" or "Powell", but I want to allow any valid value that can be passed to scipy.optimize.minimize. I don't use ChoiceField here because scipy.optimize.minimize may be extended in future.